assets/src/design-system/components/typography/link/stories/index.js

```diff
@@ -38,11 +33,7 @@ export const _default = () => (
       <Link
         key={`${presetSize}_text_link`}
         size={presetSize}
-        href="#"
-        onClick={(event) => {
```

Comment:
think this was there to just demo how tracking interacts with anchors. fine to remove, just shedding light.